app/main.py

```diff
@@ -1 +1,47 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name: str = name
+        self.skills: list[str] = []
```

Comment:
The `skills` variable is an attribute of the instance and not the class. Good job following the second point in the checklist.